File: Producto/routes-service/src/main/java/com/tagok/routes_service/dto/response/portico/PorticoTramoResponse.java
package com.tagok.routes_service.dto.response.portico;

import java.util.List;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class PorticoTramoResponse implements TollResponse
{
    private Long id;
    private String codigo;
    private String nombre;

    private double latitud;
    private double longitud;

    private String autopista;

    private List<TramoResponse> tramos;
}

File: Producto/tag-ok-app/app/src/main/java/com/tagok/app/data/dto/portico/PorticoTramoResponse.kt
package com.tagok.app.data.dto.portico

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
@SerialName("TRAMO")
data class PorticoTramoResponse(
    var id: Long,
    var codigo: String? = null,
    var nombre: String? = null,
    var latitud: Double,
    var longitud: Double,
    var autopista: String,
    var tramos: List<TramoResponse>) : TollResponse()
